lib/elixir/lib/inspect/algebra.ex

```diff
@@ -565,6 +599,10 @@ defmodule Inspect.Algebra do
     end
   end
 
+  defp ansi(color) do
+    color |> IO.ANSI.format_fragment(true)
```

Comment:
I believe it is fine because you need to explicitly opt-in for colors. You can check before setting the color options (for now).